### DISCUSSION TITLE
info su funzione countdown shelly 1v3

### DISCUSSION THREAD

**Discussion by cavodati** [indomus/forum#1992] [GENERALE - Discussioni generiche]
Ciao a tutti,
qualcuno per caso sa dirmi se nel shelley 1v3 posso impostare la funzione countdown (devo far spegnere una ventola dopo 1 ora che funziona) e se poi questa funzione è autonoma; ovvero funziona anche senza la connessione in rete wifi e senza l'accesso al cloud ? Ho provato con una meross che funziona benissimo ma deve necessariamente essere connessa ad internet. La schelley non l'ho ancora acquistata mi interessa prima sapere se questa funziona è supportata.
Grazie. Lorenzo

**Comment by Scialla** [comment]
Gli Shelly hanno l'auto off impostabile con i secondi che credi.
Funzionano offline, io uso l'auto off avendoli impostati in mqtt e quindi in locale, non uso il cloud shelly.

**Reply by cavodati** [reply]
grazie
